spec/swagger_helper.rb

```diff
@@ -0,0 +1,145 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.configure do |config|
+  # Specify a root folder where Swagger JSON files are generated
+  # NOTE: If you're using the rswag-api to serve API descriptions, you'll need
+  # to ensure that it's configured to serve Swagger from the same folder
+  config.swagger_root = Rails.root.join('modules', 'claims_api', 'app', 'swagger').to_s
```

Comment:
how does this work if another module wanted to use rswag? since this file isnt within our module

Reply:
well.... lame... i don't see a way to do this for multiple modules... this might just have to be good enough for now i guess

Reply:
ok, i think this is fine for now. it looks like we could make swagger_root root of vets-api and then define multiple modules within this file, if im reading it correctly.